.github/CODEOWNERS

```diff
@@ -5,7 +5,7 @@
 # Root
 * @smartcontractkit/foundations
 
-.changeset @smartcontractkit/releng @smartcontractkit/foundations
+.changeset @smartcontractkit/foundations
```

Comment:
@jmank88 would you be good w/ this?

Reply:
FWIW, I'd be happy with https://github.com/orgs/smartcontractkit/teams/engineers here

Reply:
We just added releng here because we needed more reviewers. How about core?

Reply:
Added core in 40a39d79777a10c455235fdfb8a6724bde413e16